server/network.go

```diff
@@ -72,6 +75,9 @@ func newNetwork(execPath string, rootDataDir string, whitelistedSubnets string,
 		nodeNames[i] = nodeName
 		cfg.NodeConfigs[i].Name = nodeName
 
+		// drop any flags received in original conf
+		cfg.NodeConfigs[i].Flags = nil
```

Comment:
Why do we do this?

Reply:
Probably not needed. I wanted to keep the previous behaviour of the network created by the server, which was to let the runner create the ports nums when executing addNode (as the node config in server does not have API port, no flag was needed).

Reply:
Will remove this. Not needed at this moment.